### ISSUE TITLE
LLM相关--如何构建一个专业大模型

### ISSUE THREAD

**Issue by winterpi** [winterpi/blog#75]
### 构建大模型的基本过程
- 预训练（pre-training， PT）
- 监督微调（supervised fine-tuning, SFT）
- 构建 Reward 打分模型（Reward Modeling, RM）
- 基于人类反馈的强化学习（Reinforcement Learning from Human Feedback，RLHF）
